data/docs/messaging-queues/confluent-kafka.mdx

```diff
@@ -0,0 +1,681 @@
+      </i>
+    </figcaption>
+</figure>
+
+On the next page **Create API key -> API key detail**, provide an appropriate name and description for the key, and click on **Create API key** button at the bottom of the page.
+
+<figure data-zoomable align="center">
+    <img
+      src="/img/docs/messaging-queues/confluent-kafka/set-api-key-detail.webp"
+      alt="Set API Key Detail"
+    />
+    <figcaption>
+      <i>
+        Set API Key Detail
+      </i>
+    </figcaption>
+</figure>
+
+On the page that follows, you will have the **Key** and **Secret** of the generate API key. Note down the key and secret as we will need it later.
+You can also download the API key and secret by clicking on **Download API key** button at the bottom of the page. Click on **Complete**.
+
+<figure data-zoomable align="center">
+    <img
+      src="/img/docs/messaging-queues/confluent-kafka/generated-api-key.webp"
+      alt="Generated API Key"
+    />
+    <figcaption>
+      <i>
+        API Key is generated
+      </i>
+    </figcaption>
+</figure>
+
+With this, we have created the Cloud API key.
+
+### Step 2: OpenTelemetry Collector Setup
+
+Install and configure OpenTelemetry for scraping the metrics from Confluent Cloud. Follow [OpenTelemetry Binary Usage in Virtual Machine](https://signoz.io/docs/tutorial/opentelemetry-binary-usage-in-virtual-machine/) guide for detailed instructions.
+
+Create a file `config.yaml`. Here, we will setup the OpenTelemetry Collector config where we will use prometheus receiver which will scrape the metrics from the Confluent Cloud Metrics API.
+
+Here are the contents of the `config.yaml` file:
+
+<Tabs>
+<TabItem value="cloud" label="SigNoz Cloud" default>
+
+```
+receivers:
+  prometheus:
+    config:
+      scrape_configs:
+        - job_name: 'confluent-cloud'
+          scrape_interval: 1m
+          scrape_timeout: 1m
+          honor_timestamps: true
+          metrics_path: "/v2/metrics/cloud/export"  # The endpoint for the metrics
+          static_configs:
+            - targets:
+              - api.telemetry.confluent.cloud  # Confluent Cloud Metrics API URL
+          scheme: https
+          basic_auth:
+            username: <Cloud API Key>
+            password: <Cloud API Secret>
+          params:
+            "resource.kafka.id":
+              - <Kafka Cluster ID, starts with `lkc-`>
+              - <Another Kafka Cluster ID, starts with `lkc-`>
+processors:
+  resource/env:
+    attributes:
+    - key: deployment.environment
+      value: prod
+      action: upsert
+  batch: {}
+exporters:
+  otlp:
+    endpoint: "ingest.{region}.signoz.cloud:443"
+    tls:
+      insecure: false
+    headers:
+      "signoz-access-token": "<SIGNOZ_INGESTION_KEY>"
+service:
+  pipelines:
+    metrics:
+      receivers: [prometheus]
+      processors: [resource/env,batch]
+      exporters: [otlp]
+```
+
+Depending on the choice of your region for SigNoz cloud, the otlp endpoint will vary according to this table.
```

Comment:
Can you remove the table below and hyperlink to https://signoz.io/docs/ingestion/signoz-cloud/overview/

This doc explains about the endpoints.